### PR TITLE
upgpkg hipify-clang 5.1.1-1

### PR DESCRIPTION
**Pull request by acxz**

### CODE REVIEW

File: hipify-clang/.SRCINFO [hunk 1]
```diff
@@ -1,14 +1,14 @@
 pkgbase = hipify-clang
 	pkgdesc = Convert CUDA to Portable C++ Code
-	pkgver = 5.0.2
+	pkgver = 5.1.1
 	pkgrel = 1
 	url = https://github.com/ROCm-Developer-Tools/HIPIFY
 	arch = x86_64
 	license = unknown
```
Comment (tpkessler) [inline hipify-clang/.SRCINFO:7]: With ROCm 5 hipify-clang is released with MIT license. https://github.com/ROCm-Developer-Tools/HIPIFY/tree/roc-5.1.x

Comment (tpkessler) [reply hipify-clang/.SRCINFO:7]: Could you please also add a line to the package() function to install the license file?

Comment (acxz) [reply hipify-clang/.SRCINFO:7]: Sure!

**Comment by tpkessler** [comment]
Could you also please add
```
install -Dm644 "$_dirname/LICENSE.txt" "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
```
after `make install` in `package()`?

**Comment by acxz** [comment]
Yes sir!

**Comment by tpkessler** [comment]
Thanks!

**Pull request merged**